deployment/network-operator/values.yaml

```diff
@@ -70,13 +70,13 @@ sriov-network-operator:
 
   # Image URIs for sriov-network-operator components
   images:
-    operator: ghcr.io/k8snetworkplumbingwg/sriov-network-operator:14bd335c17c1b4c6cb7d37c2972c05cc62cadeeb
-    sriovConfigDaemon: ghcr.io/k8snetworkplumbingwg/sriov-network-operator-config-daemon:14bd335c17c1b4c6cb7d37c2972c05cc62cadeeb
-    sriovCni: ghcr.io/k8snetworkplumbingwg/sriov-cni:v2.6.1
-    ibSriovCni: ghcr.io/k8snetworkplumbingwg/ib-sriov-cni:af8a57a2e5394c2409f8d5a5b796c0aa1ee02be0
-    sriovDevicePlugin: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin:1f1822bf0bbb25bff55190fcad861617c1a2abb7
```

Comment:
Line [156 ](https://github.com/Mellanox/network-operator/blob/master/deployment/network-operator/values.yaml#L156) should be updated too

Reply:
Updated